docs/README.md

```diff
@@ -100,6 +100,23 @@ ld -T kernel.ld -o kernel.bin kernel.o
 The flags added to the first command are there to help GCC produce could that will run in kernel space.
 The second command simply takes kernel.o and orders it as the linker script tells it to.
 
+## Contributors note
+
+After a bit of testing I found out that the entry symbol (defaults to \_start) must always be in front of anything - even includes. This is probably due to Pure64 loading pure binaries which don't store the entry points address. 
```

Comment:
Change this section to something like:

_The `_start` symbol must always appear first within flat binaries. Pure64 will call the start of the file so it must contain executable code._

Try to avoid documenting in first person for this section and the other sections. Also, I don't think it should be suggested that the entry point comes before includes. That's going to look odd. The easiest way to do it, using linker scripts, is to put it in a separate file and address that file in the linker script. This is the way we usually did it in the old examples.